test/compliance.spec.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-env mocha */
+'use strict'
+
+const testsTransport = require('libp2p-interfaces/src/transport/tests')
+const testsDiscovery = require('libp2p-interfaces/src/peer-discovery/tests')
+const multiaddr = require('multiaddr')
+
+const Stardust = require('../src')
+const { createPeer } = require('./utils')
+
+describe.skip('interface-transport compliance', () => {
```

Comment:
Why are we skipping, add a comment if it's needed?

Reply:
I have the reason for it on the PR description. But I will create an issue and add its url here

Reply:
#21